survlime/survlime_explainer.py

```diff
@@ -264,4 +274,68 @@ def solve_opt_problem(
         prob = cp.Problem(objective)
         result = prob.solve(verbose=verbose)
         cox_coefficients = b.value[:, 0]
+
+        self.computed_weights = cox_coefficients
```

Comment:
I think this is a bad practise. Attributes must be built in the constructor. What you could do is to use an input parameter `plot_weights` and, if it is True, apart from returning the weights, plot them. If it is False, then just return the weights.